daily-builds.json

```diff
@@ -1,9 +1,12 @@
 {
   "versions": [
-    "v1.7.0",
```

Comment:
We only want the 'v' here for checking out the git tag; the image tags themselves should just be the version number (as the Operator expects)